helmfile/charts/prometheus-alerts/templates/alerts/kubernetes-system-kubelet.yaml

```diff
@@ -164,12 +164,48 @@ spec:
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
 {{- end }}
-    - alert: KubeletDown
+    - alert: KubeletDownWarning
       annotations:
         description: Kubelet has disappeared from Prometheus target discovery.
         runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-kubeletdown
         summary: Target disappeared from Prometheus target discovery.
-      expr: absent(up{job="kubelet", metrics_path="/metrics"} == 1)
+      expr: up{job="kubelet", metrics_path="/metrics"} == 0
```

Comment:
Should this be upstreamed?

Reply:
Also I know that we have gotten KubeletDown alerts in the past but with this change it would also notice when pleg is unhealthy?

Reply:
I went back and tried to see if we had received this alert during kubeflapping alert and did not find one. And this should have picked it up if it worked, as the metrics are missing while the nodes is switching between ready and not ready. 
The reason the absent does not work i guess is that up is returning 0 now when it receives no metrics.

Reply:
The upstream version of this alert is "working" as intended. It should alert when ALL kubelets are unavailable. But i think it should be 1 or more.

Reply:
How about letting kubeletDown be what what it is upstream and then our kubeletLessThan will notify if single kubelets are down, and then finally we should get targetDown alerts för more than 10% of the number of kubelets are down?

Reply:
Sure we can do it like that.